src/person/views.py

```diff
@@ -59,15 +59,19 @@ def delete(self, request):
 
 
 class UserView(generics.GenericAPIView):
-    serializer_class = UserSerializer
+    serializer_class = SimpleUserSerializer
     permission_classes = api_settings.CONSUMER_PERMISSIONS
 
     def get(self, request, id):
         """Get user by id."""
         user = User.objects.filter(id=id)
         if not user:
             return failure_response("User not found.", status.HTTP_404_NOT_FOUND)
```

Comment:
`if not User.objects.filter(id=id).exists()`

Reply:
and then you should get the user like `user = User.objects.get(id=id)`

Reply:
valid